app/package.json

```diff
@@ -1,17 +1,26 @@
 {
-  "name": "example-scenes",
+  "name": "enso-ide",
   "version": "0.1.0",
-  "description": "Example Scenes",
-  "main": "index.js",
+  "description": "Enso IDE",
+  "main": "src/index.js",
   "bin": {
     "create-wasm-app": ".bin/create-wasm-app.js"
   },
   "scripts": {
     "build": "webpack --mode production",
     "build-dev": "webpack --mode development",
-    "start": "webpack-dev-server --mode development",
+    "web:dev": "webpack-dev-server --mode development",
     "start-build-time-profile": "webpack-dev-server --mode development --profile --progress",
-    "prod-server": "npx http-server -p 3000 --proxy http://localhost:3000\\? dist"
+    "web:prod": "npx http-server -p 3000 --proxy http://localhost:3000\\? static/main",
```

Comment:
As we have our own server impl now, this should either use our server impl or should be removed - using 2 server impls would bring confusion. For example, when downloading gziped content - it will work with one server but not with second one. I'm ok with removing it for now.